app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,6 @@
+$app-image-path: "/assets/static/";
+$app-font-path: "/assets/static/";
+
```

Comment:
We should be able to remove these SASS variables as they are no longer used in `_govuk_frontend_settings.scss` file in the gem.

Reply:
Thanks, good spot! I've removed it.